WatchFacePush/buildSrc/build.gradle.kts

```diff
@@ -25,11 +25,14 @@ java {
 }
 
 repositories {
-    gradlePluginPortal()
-    google()
     mavenCentral()
+    google()
+    // Note: Jitpack is required for dependencies used by the Watch Face Push Validator.
+    maven { url = uri("https://jitpack.io") }
```

Comment:
Can you filter this down on group? jitpack is pretty open, so you don't want to randomly pickup things.

https://docs.jitpack.io/

```
      maven { 
        url "https://jitpack.io" 
        content { includeGroup "com.github.username" }
      }
```